Framework/Algorithms/src/CrossCorrelate.cpp

```diff
@@ -82,13 +84,15 @@ void CrossCorrelate::init() {
   declareProperty("ReferenceSpectra", 0, mustBePositive,
                   "The Workspace Index of the spectra to correlate all other "
                   "spectra against. ");
-  // Spectra in the range [min to max] will be cross correlated to reference.
+  // Spectra in the range [min to max] will be cross correlated to referenceSpectra.
   declareProperty("WorkspaceIndexMin", 0, mustBePositive,
                   "The workspace index of the first member of the range of "
                   "spectra to cross-correlate against.");
   declareProperty("WorkspaceIndexMax", 0, mustBePositive,
                   " The workspace index of the last member of the range of "
                   "spectra to cross-correlate against.");
+  // max is .1
+  declareProperty("MaxDSpaceShift", EMPTY_DBL(), "Optional float for maximum shift to calculate (in d-spacing)");
```

Comment:
Please can you move the new parameter to the end so that it's backwards compatible for any users that have scripts that use positional arguments?

Reply:
Done. :+1: